src/modules/previewpane/README.md

```diff
@@ -1,166 +1,3 @@
-        </RegistryKey>
-        <!-- Add new previewer to preview handlers list -->
-        <RegistryKey Root="HKCU" Key="Software\Microsoft\Windows\CurrentVersion\PreviewHandlers">
-          <RegistryValue Type="string" Name="{Clsid-Guid}" Value="Name of the Previewer" />
-        </RegistryKey>
-        <!-- Add file type association for the new Previewer -->
-        <RegistryKey Root="HKCU" Key="Software\Classes\.xyz\shellex">
-          <RegistryValue Type="string" Key="{8895b1c6-b41f-4c1c-a562-0d564250836f}" Value="{Clsid-Guid}" />
-        </RegistryKey>
-      </Component>
-```
-
-### Directly registering/unregistering DLL's
-**[Important] This method of registering Preview Handler DLL's is not recommended. It could lead to registry corruption.**
-#### Registering Preview Handler
-1. Restart Visual studio as administrator. 
-2. Sign `XYZPreviewHandler` and it's dependencies. To sign an assembly in VS, follow steps given [here](https://docs.microsoft.com/en-us/dotnet/standard/assembly/sign-strong-name#create-and-sign-an-assembly-with-a-strong-name-by-using-visual-studio).
-3. Build `XYZPreviewHandler` project.
-4. Open developer command prompt from `Tools > Command Line > Developer Command Prompt`.
-5. Run following command for each nuget and project dependency to add them to Global Assembly Cache(GAC). 
-```
-gacutil -i <path to dependency>
-```
-6. Run following commands to register preview handler.
-```
-cd C:\Windows\Microsoft.NET\Framework64\4.0.x.x
-gacutil -i <path to XYZPreviewHandler.dll>
-RegAsm.exe /codebase <path to XYZPreviewHandler.dll>
-```
-7. Restart Windows Explorer process.
-
-#### Unregistering Preview Handler
-1. Run following commands in elevated developer command prompt to unregister preview handler. 
-```
-cd C:\Windows\Microsoft.NET\Framework64\4.0.x.x
-RegAsm.exe /unregister <path to XYZPreviewHandler.dll>
-gacutil -u XYZPreviewHandler
-```
-
-## Debugging
-Since in-process preview handlers run under a surrogate hosting process (prevhost.exe by default), to debug a preview handler, you need to attach the debugger to the host process. 
-1. Click on a file with registered extension to start host process.
-2. Attach debugger in Visual studio from `Debug->Attach to Process` and select `prevhost.exe` with type `Managed(version), x64`.
-
-## Managing Preview Handlers
-
-After successful integration, your preview handler should appear in the PowerToys settings UI under the `File Explorer Preview` Tab. In here you should be able to enable and disable your preview handler.
-
-<img src="../../../doc/images/preview_pane/settings-ui.png" alt="Settings UI - File Explorer Preview Tab" >
-
-In the general settings of the Settings UI, you should be able to disable and enable all the preview handlers all at once.
-
-<img src="../../../doc/images/preview_pane/general-settings.png" alt="Settings UI - General Settings Tab" >
+[File Explorer add-ons](https://aka.ms/PowerToysOverview_FileExplorerAddOns)
```

Comment:
The README is not for user documentation, it's for dev documentation, it should not be replaced entirely, we only need to update the images.

Reply:
Ah yes, I see what you mean. I have removed the images and added an updated one.